myapp/README.md

```diff
@@ -0,0 +1,242 @@
+
+        // Currently modifying allowed input fields are not supported.
+        "USER_ALLOWED_INPUT_FIELDS": [
+            "name",
+            "dob",
+            "address",
+            "description"
+        ]
+    }
+}
+```
+* Make sure you could connect to the above configured redis server.
+* ```npm start```
+* Then you will see some similar messages as follows:
+```test
+> myapp@1.0.0 start D:\nodejs\workspace\test-backend\myapp
+> node ./bin/www
+
+D:\nodejs\workspace\test-backend\myapp\bin
+```
+* Now go to ```http://localhost:3000/``` for a list of available apis for the
+Wiredcraft User Table.
+
+## Usages:
+
+---
+
+### Allowed user fields during user create/update:
+
+---
+
+| Field name | Field format | Field description | Example |
+| :-- | :-- | :-- | :-- |
+| name | Any string | user name | Ian Jiang |
+| dob | YYYY-MM-DD | user birth day | 2021-03-02 | 
+| address | JSON string | user address | ```{ longtitude: 23.0, latitude: 45.1234, description: District A Street B }``` |
+| description | Any string | user description | A car racer |
+
+---
+
+### Allowed address fields during user create/update:
+
+---
+
+| Field name | Field format | Field description | Example |
+| :-- | :-- | :-- | :-- |
+| longtitude | Float number between -180 to 180, four valid digits after decimal point | address's longtitude | -12.34, -12.3456, 85.0 |
+| latitude | Float number between -85 to 85, four valid digits after decimal point | address's latitude | -179.88, 150.1234, 95.3 |
+| description | Any string | address description | District A, Streets B|
+
+---
+
+### API descriptions:
+
+---
+
+| Function | Get all user records |
+| :-- | :-- |
+| Link | ```http://localhost:3000/users?page={page}&pageSize={pageSize}``` |
+| Method | GET |
+| Parameters | The input page and pageSize parameters should be a positive integer |
+| Return | The fetched user records will be returned as a json array |
+
+---
+
+| Function | Get a single user |
+| :-- | :-- |
+| Link | ```http://localhost:3000/users/{userId}``` |
+| Method | GET |
+| Parameters | The input userId should be a positive integer |
+| Return | The fetched user records will be returned as a json object. |
+
+---
+
+| Function | Update a single user |
+| :-- | :-- |
+| Link | ```http://localhost:3000/users/{userId}``` |
+| Method | PUT |
+| Parameters | The input userId should be a positive integer |
+| body | The body should contain a json object with the target update field and value |
+| Return | A json object will be returned to tell the user if the update is successful |
+
+---
+
+| Function | Delete a single user |
+| :-- | :-- |
+| Link | ```http://localhost:3000/users/{userId}``` |
+| Method | DELETE |
+| Parameters | The input userId should be a positive integer |
+| Return | A json object will be returned to tell the user if the update is successful |
+
+---
+
+| Function | Create a single user |
+| :-- | :-- |
+| Link | ```http://localhost:3000/users/{userId}``` |
```

Comment:
the link is not correct,
how to get the created user's userId?

Reply:
* Thank you so much for pointing out this typo in the README.md file, and it was created due to copy/paste from the description of other api.
* The actual design does not follow that description, and if the user access the home page of this application, he should be able to see the correct link for this user creation.